asset/js/widget/SearchBar.js

```diff
@@ -39,9 +39,28 @@ define(["../notjQuery"], function ($) {
             let opener = event.currentTarget;
             let editorUrl = opener.dataset.searchEditorUrl;
             let filterQueryString = this.filterInput.getQueryString();
+            let layout = document.getElementById('layout');
 
             editorUrl += (editorUrl.indexOf('?') > -1 ? '&' : '?') + filterQueryString;
 
+            // Disable pointer events to block further function calls
+            setTimeout(function () {
+                opener.style.pointerEvents = 'none';
+            }, 20);
+
+            let observer = new MutationObserver(resetPointerEvents);
+            function resetPointerEvents(mutations) {
```

Comment:
Please don't use a standard function here. It's weird that you reference it already the line beforehand and it's this way bound to the function scope. I don't know if that's reliable enough in terms of garbage collection.

Use a closure (`() => {...}`) instead.